### PR TITLE
Fix admin by removing "reverse" from module scope (#224)

### LINKED ISSUE #224: Unable to access admin site because of module-level reverse
During testing in QA, we discovered a bug that prevented us from accessing the `admin` site. This traceback was visible in the backend:

```
django.urls.exceptions.NoReverseMatch: Reverse for 'app_list' not found. 'app_list' is not a valid view function or pattern name.
```

After investigating, @jlost and I concluded that the issue was caused by the usage of `reverse` in `officehours_api/notifications.py`, specifically the use of `reverse` at the module level in `PREF_URL`, introduced by PR #210.

### PR DESCRIPTION
**Pull request by jlost**
Closes #224

### CODE REVIEW

File: src/officehours_api/notifications.py [hunk 1]
```diff
@@ -15,11 +15,14 @@
...
 
 DOMAIN = Site.objects.get_current().domain
-PREF_URL = f"{DOMAIN}{reverse('preferences')}"
-ADDENDUM = (
-    f"\n\nYou opted in to receive these texts from U-M. "
-    f"Opt out at {PREF_URL}"
-)
+
+
+def build_addendum():
```
Comment (ssciolla) [inline src/officehours_api/notifications.py:20]: Would it be worth adding a comment or docstring explaining why we did this?

Comment (ssciolla) [reply src/officehours_api/notifications.py:20]: We both missed this initially, so someone else might not understand pick this up either.

**Review by ssciolla — COMMENTED** [review]
LGTM! Just the one suggestion about the comment/explanation. I reproduced the error locally, and then confirmed that this fixed it. Thanks for cleaning up a bug I introduced!

**Review by ssciolla — APPROVED** [review]
Sorry, I meant to approve.

File: src/officehours_api/notifications.py (hunk 1 above)
Comment (jlost) [reply src/officehours_api/notifications.py:21]: Good idea, added a comment.

**Pull request merged**